src/docs/Components/Hero/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hero'
+order: 1
+description: 'A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the  attention of the user on a specific message.'
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the attention of the user on a specific message.
+
+## Variations
+
+We want to provide as many cases as possible so we created three different versions of heroes.
+
+### Title only hero
+
+This is the most minimal hero case you will find in Mozaïc Design System. This version is best used when you need to introduce a page with just an image and a title on top of it.
+
+--title only preview--
+
+### Title with text hero
+
+This is the most common hero case we recommend to use. This version is made of a headline, a text content and a background image.
+
+--title with text preview--
+
+### Title with text and boxed content
+
+Sometimes, you need to take some content apart of the overall page layout. That is why we provide a hero version with a boxed content either centered under the text content or aligned to the right.
```

Comment:
- Replace `Sometimes, ...` by `In specific cases, ...`
- ~~...you need to take some content apart of the overall page layout.~~ → ...you need to take some content apart of the overall page layout, for example, to put forward a feature.